examples/regular_polygons.py

```diff
@@ -0,0 +1,106 @@
+"""
+This is an example script implementing
+the method normal_polygon() from the class Polygon
+for the Pygame community.
+
+Author: @newpaxonian
+"""
+
+import random
+import sys
+
+import pygame as pg
+import geometry
+
+DARK_GREY = (10, 10, 10)
+SIDES = [3, 4, 5, 6, 7, 8, 9, 10, 11, 12]
+
+
+class Game:
+    """
+    Class Game: contains the main game loop and the methods
+    """
+
+    def __init__(self):
+        pg.init()
+        self.display = pg.display
+        self.screen = self.display.set_mode((640, 480))
+        self.clock = pg.time.Clock()
+        self.radius = 64
+        self.toggle_mouse_button = False
+        self.polygons = []
+
+    def events(self):
+        """
+        Method events: handles the events
+        """
+        for event in pg.event.get():
+            if event.type == pg.QUIT or \
+                    (event.type == pg.KEYDOWN and event.key == pg.K_ESCAPE):
+                print("\nQuitting program...")
+                pg.quit()
+                sys.exit()
+            if pg.MOUSEBUTTONDOWN and any(pg.mouse.get_pressed()):
+                if self.toggle_mouse_button:
```

Comment:
You can condense these four lines to just `self.toggle_mouse_button = not self.toggle_mouse_button`

Reply:
ok, makes sense